classes/class-wc-rest-connect-address-validation-controller.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_REST_Connect_Address_Validation_Controller' ) ) {
+	return;
+}
+
+class WC_REST_Connect_Address_Validation_Controller extends WP_REST_Controller {
```

Comment:
But... (sorry)... could we be consistent with using validation or normalization?